.gitignore

```diff
@@ -13,3 +13,5 @@ coverage.html
 .glide
 .scBuildImage
 .init
+.generate*
+**/zz*.go
```

Comment:
None of the reasons for us *having* to check in our code apply to our repo yet - as far as I can tell. And I'm all for checking them in when we *have to*.  But, given @thockin's statement about trying to remove it from the main repo, it seems to me that it might actually be a benefit for us to be a guinea pig in this area - let us be the a testbed for the rest of Kube because if we find that we can't simplify the process then it may not be worth the effort for the main repo to try.  On the other hand, perhaps something can be learned from our trailblazing here - perhaps not the dream goal of not checking it in at all, but maybe we'll land on a middle group position. I would just prefer to not toss in the towel w/o trying.